src/main/java/com/gregtechceu/gtceu/api/data/worldgen/generator/indicators/SurfaceIndicatorGenerator.java

```diff
@@ -48,8 +48,8 @@ public class SurfaceIndicatorGenerator extends IndicatorGenerator {
     public static final Codec<SurfaceIndicatorGenerator> CODEC = RecordCodecBuilder.create(instance -> instance.group(
             Codec.either(BlockState.CODEC, GTCEuAPI.materialManager.codec()).fieldOf("block").forGetter(ext -> ext.block),
             IntProvider.codec(1, 32).fieldOf("radius").forGetter(ext -> ext.radius),
-            FloatProvider.codec(0.0f, 1.0f).fieldOf("density").forGetter(ext -> ext.density),
-            StringRepresentable.fromEnum(IndicatorPlacement::values).fieldOf("placement").forGetter(ext -> ext.placement)
+            FloatProvider.codec(0.0f, 2.0f).fieldOf("density").forGetter(ext -> ext.density),
```

Comment:
Anything above 1 wouldn't really do anything for the density, as it's basically used for a comparison against a random value between 0 and 1 (see line 122 and 132)

Reply:
see commit message.
had to change it so that a constant number for density is valid because for some reason ConstantFloat's max is x+1 instead of x.

Reply:
Huh wtf, certified minecraft moment right there...  
Should probably have a comment explaining it though, imo